src/main/java/mate/academy/Main.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy;
+
+import core.basesyntax.Lottery;
+
+public class Main {
+    public static void main(String[] args) {
+        // create three balls using class Lottery and print information about them in console
+        Lottery[] balls = new Lottery[3];
```

Comment:
no need to create 3 lotteries to generate 3 balls, also no need to create an array
```suggestion
        Lottery lottery = new Lottery();
```